sunbird-cassandra-migration/cassandra-migration/src/main/resources/db/migration/cassandra/V1.93_cassandra.cql

```diff
@@ -0,0 +1 @@
+ALTER TABLE sunbird.otp ADD attemptedCount int;
```

Comment:
attemptedCount property need to be added inside cassandra-column.properties file , to read it back.

Reply:
already there.